cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -1407,6 +1428,23 @@ fn x64_get_regs(inst: &Inst, collector: &mut RegUsageCollector) {
             src.get_regs_as_uses(collector);
             collector.add_mod(*dst);
         }
+        Inst::XmmRmRImm { src, dst, op, .. } => {
+            // In certain cases, instructions of this format can convert an XMM register into a
+            // define (e.g. an equality comparison); this extra logic is necessary to inform the
+            // registry allocator of a different register usage.
+            let is_define = if let RegMem::Reg { reg } = src {
```

Comment:
light suggestion: we use `def` for `definition` / `define` in misc places in the code base, so for consistency, it'd be nice to use it here too.